src/sinks/aws_sqs/sink.rs

```diff
@@ -50,7 +53,18 @@ impl SqsSink {
 
         let sink = input
             .request_builder(request_builder_concurrency_limit, self.request_builder)
-            .filter_map(|req| async move { req.ok() })
+            .filter_map(|req| async move {
+                match req {
+                    Err(error) => {
+                        emit!(SinkRequestBuildError {
+                            name: SqsSinkConfig::NAME,
+                            error,
+                        });
+                        None
+                    }
+                    Ok(msg) => Some(msg),
+                }
```

Comment:
Not important, but this may be a little neater?

```suggestion
                req.map_err(|error| {
                        emit!(SinkRequestBuildError {
                            name: SqsSinkConfig::NAME,
                            error,
                        });
                }).ok()
```